Mathlib.lean

```diff
@@ -5590,6 +5590,7 @@ import Mathlib.Tactic.HaveI
 import Mathlib.Tactic.HigherOrder
 import Mathlib.Tactic.Hint
 import Mathlib.Tactic.ITauto
+import Mathlib.Tactic.ImportDiff
```

Comment:
```suggestion
import Mathlib.Util.ImportDiff
```
I think that this would be a better location for this function.  I may have misplaced functions between `Util` and `Tactic`, but this feels more like a `Util` than a `Tactic`.